website/src/theme/MDXComponents/VideoContainer.js

```diff
@@ -36,9 +36,8 @@ export default function VideoContainer({ title, list }) {
       <Grid container spacing={4}>
         {videos}
       </Grid>
-      <Divider style={{ paddingTop: '1.8em' }} variant='fullWidth'></Divider>
     </div>
   )
 }
 
-VideoContainer.isMDXComponent = true
+VideoContainer.isMDXComponent = true
```

Comment:
same here, was it needed?

Reply:
The line is inlcuded in the current development branch, but I just checked project still builds and looks normal after the removal.